modules/ROOT/pages/tutorial-manage-an-api.adoc

```diff
@@ -8,42 +8,55 @@ In this procedure, you configure and apply a rate limiting SLA-based policy to a
 * A tier that limits requests to three per minute, with no approval required
 * A tier that limits requests to five per minute, with API Versions Owner approval required
 
-When an app attempts to consume the protected API resource, the rate limiting SLA-based policy is enforced. Because this policy is client ID-based, app requests must include the expected client ID. The client secret is optional.
+When an app attempts to consume the protected API resource, the rate limiting SLA-based policy is enforced. Because this policy is client ID-based, app requests must include the expected client credentials. The procedure demonstrates including client credentials as headers, which is more secure than including the credentials as query parameters.
 
+To configure and apply a rate limiting SLA-based policy to an API:
+
+. <<add-raml,Add the Required RAML Snippet>>.
+. <<add-sla-tier,Add the SLA Tier>>.
+. <<apply-policy,Apply the Policy>>.
+. <<register-deploy,Register and Deploy>>.
+
+[[add-raml]]
 == Add the Required RAML Snippet
 
 SLA-based rate limiting requires adding a RAML or OAS snippet to your API. This procedure demonstrates adding a RAML snippet.
 
-. Specify the client ID and secret as query parameters. 
+. Specify the client ID and secret as headers. 
 +
-Add a section called `traits:` at the RAML root level to define query parameters:
+Add a section called `traits:` at the RAML root level to define headers:
 +
 [source,yaml,linenums]
 ----
 traits:
-  - client-id-required:
-      queryParameters:
-       client_id:
+  client-id-required:
+    headers:
+      client_id:
         type: string
       client_secret:
         type: string
 ----
 
-. Add the `client-id-required` trait to every method that requires these query parameters:
+. Add the `client-id-required` trait to every method that requires these headers:
 +
 [source,yaml,linenums]
 ----
 /users:
   get:
     is: [client-id-required]
-    description: Gets a list of JSONPlaceholder users.
 ----
 
+. xref:design-center::design-publish.adoc[Publish the API to Exchange]
+
+[[add-sla-tier]]
 == Add the SLA Tier
 
 With the required RAML snippet added to your RAML API, you now add the SLA tier.
 
-. From API Manager, navigate to *API Administration* and select an API instance.
+. From API Manager, navigate to *API Administration* and select the API instance.
+. Click *Actions*.
+. Click *Change API Specification*. 
```

Comment:
Remove this step if you apply the above update.

Reply:
Fixed!